src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/shared/helpers/terraform/adf_terraform.sh

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env bash
+PATH=$PATH:$(pwd)
+export PATH
+CURRENT=$(pwd)
+terraform --version 
+echo "Terraform stage: $TF_STAGE"
+
+tfinit(){
+    # retrieve regional S3 bucket name from parameter store
+    S3_BUCKET_REGION_NAME=$(aws ssm get-parameter --name /cross_region/s3_regional_bucket/"$AWS_REGION" --region "$AWS_DEFAULT_REGION" | jq .Parameter.Value | sed s/\"//g)
+    mkdir -p "$CURRENT"/tmp/"$TF_VAR_TARGET_ACCOUNT_ID"-"$AWS_REGION"
```

Comment:
Could you update the use of quotes on this line?

Like this:
```suggestion
    mkdir -p "${CURRENT}/tmp/${TF_VAR_TARGET_ACCOUNT_ID}-${AWS_REGION}"
```

Reply:
Could you process the other quotes in this file like that too?